### DISCUSSION TITLE
Choice of cheap probe for plus starter

### DISCUSSION THREAD

**Discussion by altif** [PlusToolkit/PlusLib#980] [Q&A]
![Screenshot_2022-09-04-12-23-13-57_3c841798971d89a92069939d312bec3c](https://user-images.githubusercontent.com/33426958/188298743-bfe13087-8db0-4d76-90ee-8dcd2f9b710c.jpg)
Hi, would this kind if entry level cn probe do the job? Or should I go to Clarius? Or other suggestion?
Tgank you

**Comment by adamrankin** [comment]
Could you give us a bit more information about the probe? Does it have the ability to stream to a pc? Is there an api?

**Reply by altif** [reply]
Hi, I this is sonostar oldest model, I sent them a message I'll report if they answer

https://fr.sonoeye.com/single-head-ultrasound-series/59633832.html

**Comment by adamrankin** [comment]
It seems they stream to windows, can you confirm?
